### PR TITLE
refactor: reuse common ui dialogs and message for mobile and LayerUI

### PR DESCRIPTION
**Pull request by ad1992**

### CODE REVIEW

**Comment by ad1992** [comment]
should JSONExport and Image export dialog be part of `CommonUIDialogs` as well ?

File: src/components/LayerUI.tsx [hunk 1]
```diff
@@ -381,40 +378,13 @@ const LayerUI = ({
...
-          }
-        />
-      )}
-    </>
-  );
-
   return device.isMobile ? (
     <>
-      {dialogs}
+      <CommonUIDialogs
```
Comment (ad1992) [inline src/components/LayerUI.tsx:383]: any better name ?

Comment (dwelle) [reply src/components/LayerUI.tsx:383]: there's not much commonality so can't say :)

Comment (dwelle) [reply src/components/LayerUI.tsx:383]: maybe split it into `<UIMessages/>`, `<HelpDialog/>`, and `<PasteDialog/>`?

**Review by dwelle — APPROVED** [review]

**Pull request merged**